test_app/tests/lib/redis/test_client.py

```diff
@@ -67,8 +81,9 @@ def test_redis_client_confirm_connect_does_not_change_options():
     ],
 )
 def test_redis_client_location_parsing(url, host, port, username, password, db, raises, socket):
-    redis_cache = RedisCache(url, {})
-    client = RedisClient(url, {}, redis_cache)
+    args = {'OPTIONS': {'CLIENT_CLASS_KWARGS': {'mode': 'standalone'}}}
```

Comment:
Looks like no mode specified should still result in standalone since it's defaulted that way earlier, so maybe test both here?